ambry-store/src/main/java/com.github.ambry.store/HardDeleter.java

```diff
@@ -95,6 +95,7 @@
   private final AtomicBoolean paused = new AtomicBoolean(false);
   private final ReentrantLock hardDeleteLock = new ReentrantLock();
   private final Condition pauseCondition = hardDeleteLock.newCondition();
+  static final String HARD_DELETE_OPS_JOB_NAME = "hardDeleteOps";
```

Comment:
`hard_deleter_ops`? (or something that starts with `hard_deleter_`

Reply:
```static final String HARD_DELETE_CLEANUP_JOB_NAME = "hard_deleter_cleanup";```

Reply:
done.